workflow-core/src/main/java/com/squareup/workflow/Worker.kt

```diff
@@ -117,6 +122,14 @@ import kotlin.reflect.typeOf
  */
 interface Worker<out OutputT> {
 
+  /**
+   * Should be overridden in subclasses that have their own type parameters, to allow the runtime to
+   * make use of the value of those type parameters to compare workers. Two workers of the same
+   * concrete [Worker] class will be considered equivalent if and only if their [outputType]s are
+   * also equivalent. If this property returns null, the worker will be treated as a `Worker<*>`.
+   */
+  val outputType: KType? get() = null
```

Comment:
This was originally intended to be used to create the workflow identifier without reflecting on the worker itself, but that didn't work out since we're forced to just use `typeOf<W>()` (where `W` is the reified type of the worker) in `runningWorker` to get the worker type. We can't create the `KType` from just the worker `KClass` and this `outputType` (e.g. do something like `worker::class.createType(KTypeProjection.covariant(worker.outputType))`) because we don't have access to the full reflection library from the core module. 

This should be removed since it's not used.